schema/constants.go

```diff
@@ -36,6 +36,7 @@ const (
 	CreateTableCouponsSQL                   = "create table coupons (slug text, code text, hash text);"
 	CreateIndexCouponsSQL                   = "create index index_coupons on coupons (slug);"
 	CreateTableModeratedStoresSQL           = "create table moderatedstores (peerID text primary key not null);"
+	CreateOrderMessagesSQL                  = "create table order_messages (messageID text primary key not null, orderID text, message_type integer, message blob, peerID text, url text, acknowledged bool, tries integer, created_at integer, updated_at integer);"
```

Comment:
What is `url` for?

Reply:
It is for IPFS URL. We might use the IPFS url (in future) or message blob to resend a message.